.drone.yml

```diff
@@ -24,13 +24,25 @@ services:
       - EPOETRY_AUTH_CERT_PATH=""
       - EPOETRY_AUTH_CERT_PASSWORD=""
       - EPOETRY_AUTH_EU_LOGIN_BASEPATH=""
+      - CDT_TOKEN_API_ENDPOINT=https://example.com/token
```

Comment:
not sure when these example.com values will come into play, can we remove them and add anything only when needed